src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -189,25 +217,31 @@ def __disable_buttons(self):
         Colors the clicked button yellow, the lower buttons are colored green.
         All other buttons keep their original color
         """
-        counter = 0
         starting_index = self._starting_point.get_order()
 
-        for button in self.buttons:
+        for i, button in enumerate(self.buttons):
             button.config(state="disable")
-            if counter == starting_index:
-                button.configure(bg_color="yellow")
-            elif counter < starting_index:
-                button.configure(bg_color="green")
-            counter += 1
+
+            if i == starting_index:
+                button.configure(fg_color=button_constants_i.ButtonConstants.BUTTON_FG_COLOR_YELLOW.value)
+            elif i < starting_index:
+                button.configure(fg_color=button_constants_i.ButtonConstants.BUTTON_FG_COLOR_GREEN.value)
 
     def __activate_buttons(self):
         """
         Reactivates the buttons of the class and makes them clickable again
         """
         for button in self.buttons:
-            button.configure(state="enable")
+            button.configure(state="enable",
```

Comment:
pretty sure it say "normal" in the customtkinter github for activating a button

Reply:
Yes, but if you want to reactivate the buttons to restart the calculation you need a function to enable the buttons again